helm/gomodel/templates/pdb.yaml

```
@@ -0,0 +1,18 @@
+{{- if .Values.podDisruptionBudget.enabled }}
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ include "gomodel.fullname" . }}
+  labels:
+    {{- include "gomodel.labels" . | nindent 4 }}
+spec:
+  {{- if .Values.podDisruptionBudget.minAvailable }}
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)